shared/cloud/ess-getting-started.asciidoc

```diff
@@ -6,12 +6,14 @@
 
 There's no faster way to get started than with our hosted {ess} on Elastic Cloud:
 
-. {ess-trial}[Get a free trial]. 
+. {ess-trial}[Get a free trial].
 
 . Log into {ess-console}[Elastic Cloud].
 
-. Click *Create deployment* and give your deployment a name.
-. Optional: The default options are great for getting started, but you can add more features by clicking *Customize deployment*.
-. Click *Create deployment* and copy down the password for the `elastic` user and Cloud ID information.
+. Click *Create deployment*.
+
+. Select a deployment type and give your deployment a name.
```

Comment:
```suggestion
. Select a solution and give your deployment a name.
```

Milestone 45 brought with it support for our 3+1 solutions. Technically, it's not wrong to say "deployment type" but the UI does refer to solutions. It might be helpful to stick with consistent terminology. 

![image](https://user-images.githubusercontent.com/15148011/94619849-6e915180-0262-11eb-9389-05f13b5c0d57.png)

Reply:
Great point!